site/content/blog/2021-07-01-whats-new-in-svelte-july-2021.md

```diff
@@ -0,0 +1,56 @@
+---
+title: What's new in Svelte: July 2021
+description: Keeping cool with fixes, TypeScript tooling and tonnes of new features
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+As the northern hemisphere heats up, Svelte has stayed cool with lots of performance and bug fixes, better TypeScript support, and lots of new components & tools from around the ecosystem. Let's take a peek 👀
+
+## New in SvelteKit
+- `adapter-node` now precompresses assets using gzip & brotli ([#1693](https://github.com/sveltejs/kit/pull/1693))
+- Support for TypeScript transpilation has been added to the svelte package tooling ([#1633](https://github.com/sveltejs/kit/pull/1633))
+
+
+## Features & bug fixes from around svelte/*
+- [Svelte 3.38.3](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#3383) (released June 22) includes a bunch performance and bug fixes - including hydration optimizations, `this` preservation in bubbled events, and more!
+- The latest language tools releases added support for prop renaming from outside of a component, postcss syntax grammar, `.d.ts` file outputs in `svelte2tsx` and better support for TypeScript features across the board!
+- `svelte-scroller` got some quality-of-life fixes in 2.0.7 - fixing an initial width bug and updating its `index` more conservatively
+
+---
+
+## Community Showcase
+
```

Comment:
[The sveltejs/community repository](https://github.com/sveltejs/community) was archived in favor of [the Svelte Society site](https://sveltesociety.dev/). The Svelte Society website is currently in the process of being [rewritten in SvelteKit](https://github.com/svelte-society/sveltesociety-2021). See [the list of open issues](https://github.com/svelte-society/sveltesociety-2021/issues) if you'd like to contribute.

As part of archiving the old `sveltejs/community` repo, we decided that the list of company logos displayed on [the Svelte homepage](https://svelte.dev/) had become to large to display in a reasonable fashion and dramatically reduced the number of logos displayed to only the most prominent companies. If you work at a well-known company, [please feel free to send a PR](https://github.com/sveltejs/svelte/blob/master/site/src/routes/_components/WhosUsingSvelte.js) to add your company's logo.

Reply:
Tangent: I really want to chase down some of those companies...